t/24-worker-webui-connection.t

```diff
@@ -358,7 +358,7 @@ subtest 'send status' => sub {
     $client->send_status();
     is_deeply($ws->sent_messages, [{json => {fake_status => 1, reason => 'some error'}}], 'status sent')
       or always_explain $ws->sent_messages;
-    combined_like { Mojo::IOLoop->one_tick } qr/some error.*checking again/, 'error logged in callback';
+    combined_like { $client->send_status_delayed } qr/some error.*checking again/, 'error logged in callback';
```

Comment:
Oh found it! (https://github.com/os-autoinst/openQA/pull/6358/files#r2048763363)